var/spack/repos/builtin/packages/py-rtree/package.py

```diff
@@ -25,3 +25,6 @@ def setup_build_environment(self, env):
 
     def setup_run_environment(self, env):
         self.setup_build_environment(env)
+
+    def setup_dependent_run_environment(self, env, dependent_spec):
+        self.setup_build_environment(env)
```

Comment:
```suggestion
        self.setup_build_environment(env)
```

Reply:
Quickly fixed that myself… in general, files should end with a newline, which some editors don't do :-/

Reply:
@matz-e Thanks for the fix. I usually run flake8 before submitting a PR, but this time I was obviously sloppy.

Reply:
It was a simple change :)